map/extrapolation_benchmark/extrapolation_benchmark.cpp

```diff
@@ -0,0 +1,331 @@
+#include "map/extrapolation/extrapolator.hpp"
+
+#include "routing/base/followed_polyline.hpp"
+
+#include "geometry/distance_on_sphere.hpp"
+#include "geometry/latlon.hpp"
+#include "geometry/mercator.hpp"
+#include "geometry/polyline2d.hpp"
+
+#include "platform/location.hpp"
+
+#include "base/logging.hpp"
+#include "base/string_utils.hpp"
+
+#include <cmath>
+#include <cstdint>
+#include <fstream>
+#include <limits>
+#include <sstream>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "3party/gflags/src/gflags/gflags_declare.h"
+#include "3party/gflags/src/gflags/gflags.h"
+
+// This tool is written to estimate quality of location extrapolation. To launch the benchmark
+// you need tracks in csv file with the format described below. To generate the csv file
+// you need to go through following steps:
+// * take logs form Trafim project production in gz files
```

Comment:
s/form/from/
What is the "Trafim project production"?

Reply:
Fixed.

"trafin" (but not Trafim) - traffic income. Fixed. It's the name of the server where logs should be taken for this benchmark.